src/react/jsx/atc.js

```diff
@@ -18,7 +18,7 @@ var ShapingPanel = require('./shaping');
 var Atc = React.createClass({
   getInitialState: function() {
     return {
-      client: new AtcRestClient(this.props.primary, this.props.secondary, this.props.endpoint),
+      client: new AtcRestClient(),
```

Comment:
yeah, for the sake of this diff, I was assuming that the default was fine.... mainly if we want to separate the ui from the api.... but yes, there may be situation where it does not work.